TM1py/Services/ProcessService.py

```diff
@@ -82,6 +82,48 @@ def get_all_names(self, **kwargs) -> List[str]:
         processes = list(process['Name'] for process in response.json()['value'])
         return processes
 
+    def search_string_in_body(self, search_string: str, **kwargs) -> List[str]:
```

Comment:
Actually, we are not looking the full body of a process (parameters, variables, etc.) but only at the code tabs.

Perhaps it would be more precise to call the function something like `search_string_in_code_tabs`

Reply:
yes good point, i can update to `search_string_in_code` or possibly update the function to search the full body of the process

Reply:
updated: https://github.com/cubewise-code/tm1py/pull/658/commits/3a372f9acc16215e47fbbb7d82f4c39d56799a8b